App/frontend/src/components/components/inputField.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
okay theo, this is my first suggestion, u don't need to make a component for every button, if u think about it it would be pointless since it's going to use more space, ur writing a lot unnecessary code and the point of making a component like this is that it will be reusable

so, make a button component that u can edit with props, there are some tutorials about how to do that.